### ISSUE TITLE
crypto trading?

### ISSUE THREAD

**Issue by flyboynyc** [siropkin/robinhood-ai-trading-bot#37]
Hello, does this bot also do crypto trading?

**Comment by Anitasmith0072** [comment]
Yes, this bot trades. But before you get started. Visit our live support page to initiate a chat with an agent for further information and enquires on how it works via [Robinhood-ai-trading live chat](https://generalwebfixes.web.app).
Regards

**Comment by siropkin** [comment]
@flyboynyc Hi! No, it's not right now but I can add it in the future if you want

**Comment by flyboynyc** [comment]
> @flyboynyc Hi! No, it's not right now but I can add it in the future if you want

PLEASE! I can donate to you as well if u can add feature and also make a video of the setup please. I have done everything except I cant generate a API key in the robinhood app. If you can do this, we can work together and make money. Thanks in advance!

**Comment by siropkin** [comment]
Give me some time to check...

**Comment by Michaelthen00b** [comment]
> > @flyboynyc Hi! No, it's not right now but I can add it in the future if you want
> 
> PLEASE! I can donate to you as well if u can add feature and also make a video of the setup please. I have done everything except I cant generate a API key in the robinhood app. I want to strictly trade only crypto. If you can do this, we can work together and make money. Thanks in advance!

You do not need to generate an API key from the robinhood app, just your user name and password are reqired. You only need an API key for open ai.
